public/src/utils/requests.tsx

```diff
@@ -108,6 +108,13 @@ export function fetchSupportFrontendSettings(
   return fetchSettings(`/support-frontend/${settingsType}`);
 }
 
+export function fetchCampaignTests(
+  campaign: string,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
If you return `Promise<Test[]>` here then the eslint-disable isn't necessary

Reply:
Done